src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,39 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(health, name) {
```

Comment:
I'm not sure if your constructor needs to take health as an argument, because you set its value to 100 as default. Overall well done!

Reply:
yep, remove this param
```suggestion
  constructor(name) {
```